### PR TITLE
fix: changed displaynames for DCs, added a check for the subscription…

### PR DESCRIPTION
**Pull request by AnaBil-k**
… log file

### Description

- Changed the display names for DCs in the ibm_catalog.json,
- added a check condition for the subscription log file.
### Release required?

- [x] No release
- [ ] Patch release (`x.x.X`)
- [ ] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### Merge actions for mergers

- When merging, use a relevant [conventional commit](https://www.conventionalcommits.org/) message that is based on the PR contents and any release notes provided by the PR author. The commit message determines whether a new version of the module is needed, and if so, which semver increment to use (major, minor, or patch).
- Merge by using "Squash and merge".

### CODE REVIEW

File: submodules/configure_network_services/templates/services_init.sh.tftpl [hunk 1]
```diff
@@ -154,6 +154,12 @@ register_os() {
         done
       fi
 
+     ##### check if the subscription log file /var/log/powervs-fls.log exists
+      if [ ! -e "$SUBSCRIPTION_LOG" ]; then
```
Comment (surajsbharadwaj) [inline submodules/configure_network_services/templates/services_init.sh.tftpl:158]: in print change it to : Subscription log $SUBSCRIPTION_LOG ,
 and this block should come just before while loop at the bottom inside the ppc64le

**Review by surajsbharadwaj — CHANGES_REQUESTED** [review]

File: submodules/configure_network_services/templates/services_init.sh.tftpl (hunk 1 above)
Comment (AnaBil-k) [reply submodules/configure_network_services/templates/services_init.sh.tftpl:158]: You mean inside of this block:

 ```
##### check if the system is a ppc64le processor
      if [[ "$ARCH" == "ppc64le" ]]; then

        if [ ! -e "$README" ]; then
          print_out "PER enabled DC. Registration was done automatically."
        else
          #will run the register command
          
          ##### check if the subscription log file /var/log/powervs-fls.log exists
        if [ ! -e "$SUBSCRIPTION_LOG" ]; then
          print_out "Subscription log $SUBSCRIPTION_LOG is missing.Cannot perform OS Registration.OS Registration failed."
          exit 1
        fi

          suseconnect_register
        fi
```

Comment (surajsbharadwaj) [reply submodules/configure_network_services/templates/services_init.sh.tftpl:158]: https://github.com/terraform-ibm-modules/terraform-ibm-powervs-infrastructure/blob/bc0bea3d06616c1ea3082a5382f812adfc47bc66/submodules/configure_network_services/templates/services_init.sh.tftpl#L235
add sleep 60 and then insert the block and then while loop follows.

Testing is required .
Deploy an instance. Then login to the OS. Unregister it.
Delete "/usr/share/powervs-fls/powervs-fls-readme.md" and  "/var/log/powervs-fls.log" files. 
Run OS registration the file is in terraform_scripts dir :  services_init register_os 

It should out print:  "Subscription log $SUBSCRIPTION_LOG is missing.Cannot perform OS Registration.OS Registration failed."

Please verify

**Review by surajsbharadwaj — CHANGES_REQUESTED** [review]
Please fix all the comments

**Review by surajsbharadwaj — APPROVED** [review]
Looks great. Thanks

**Pull request merged**